### ISSUE TITLE
[FEAT] Option to pin tray items, change languages, scroll implementation for audio devices, separate icons for programs and toolbar element offset.

### ISSUE THREAD

**Issue by Etheirystech** [eythaann/Seelen-UI#198]
**Is your feature request related to a problem? Please describe.**

**Pin tray items:**
I need a way to pin certain system tray items
![image](https://github.com/user-attachments/assets/5767b5eb-c0f5-40f8-8bfd-07016937f9db)

**Change languages:**
Currently there is no option change or see the current language being used.

**Scroll for audio devices:**
Because there is no scroll for audio devices some devices wont even show up after the element covers the whole screen.
![image](https://github.com/user-attachments/assets/9d5780aa-6a56-49c0-a78e-6ca7be9f43c8)

**Describe the solution you'd like**

**Pin Tray items:**
They should be located besides other tray items
![image](https://github.com/user-attachments/assets/40ac1776-0589-4290-863b-2d5143bceb74)

**Change languages:**
Something similar to what windows currently has.

**Scroll for audio devices:**
I like the way that the current audio element looks like, so I would say that it would be good to give the user the option
to select how many devices they would like to display, and the rest of the devices would be visible after they scroll.

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by Etheirystech** [comment]
**Is your feature request related to a problem? Please describe.**

**Separate icons:**
Currently multiple windows would be combined into one icon.
![image](https://github.com/user-attachments/assets/4250b336-23f8-4dcb-8c18-c5c9410bf2ad)

**Describe the solution you'd like**

**Separate icons:**
Different profiles/windows should have different icons.
![image](https://github.com/user-attachments/assets/e05590ae-a5dc-43eb-886a-729758e4b18d)

It would also be nice if you could added a feature where the user would be able to change an icon of any program through the context menu for an icon that is pined.

**Comment by Etheirystech** [comment]
**Is your feature request related to a problem? Please describe.**

**Toolbar offset:**
Currently there is no way to offset the left and right elements of the toolbar.
![image](https://github.com/user-attachments/assets/bf19f3dd-3159-4470-b9e1-12acf3629ba5)
![image](https://github.com/user-attachments/assets/5e65f0f5-eb28-4f4f-b094-b744d37afb0c)

**Describe the solution you'd like**

**Toolbar offset:**

Basically there should be a menu where the user could specify the amount of padding that they want to add for left and right elements.
This way they can move those elements closer to the middle of the monitor if they are using an ultrawide monitor or a multi monitor setup with Nvidia surround.

**Comment by eythaann** [comment]
wow, how many audio devices you have, yeah I will add a collapse box instead an scroll, or probably both.

**Comment by Etheirystech** [comment]
Great!

By the way awesome job so far.

I really needed something like this because windows UI is just loves to break in Nvidia surround.

And when you add the padding for toolbar elements that's gonna be amazing.

**Comment by gbellini79** [comment]
The language bar or something similar integrated in the Fancy Toolbar would be very useful.
I have a US keyboard but writing often in italian which uses accented characters I need to switch frequently from a keyboard layout to another and without a visual indicator the only way to know which one is currently active is to write something.
At the moment I have the native Windows language bar floating around.

**Comment by eythaann** [comment]
On this ticket I will only fix the devices overflow, Keyboard Language module will come on other ticket, the same for Pin Tray items.
Also I think the bug related with chrome profiles was already fixed, please test it on v2.0.5 @Etheirystech  if it doesn't work yet, create a issue about that (only for chrome profiles avoid adding many thing on one issue)

**Comment by Etheirystech** [comment]
> On this ticket I will only fix the devices overflow, Keyboard Language module will come on other ticket, the same for Pin Tray items. Also I think the bug related with chrome profiles was already fixed, please test it on v2.0.5 @Etheirystech if it doesn't work yet, create a issue about that (only for chrome profiles avoid adding many thing on one issue)

Still doesn't work on 2.0.4, I'll try again on 2.0.5.

Regarding the Keyboard Language, Pin Tray items and Toolbar offset, should I create separate ticket for those?

**Comment by eythaann** [comment]
there are already issues created by other users so just wait for the resolution of them

**Comment by Etheirystech** [comment]
> On this ticket I will only fix the devices overflow, Keyboard Language module will come on other ticket, the same for Pin Tray items. Also I think the bug related with chrome profiles was already fixed, please test it on v2.0.5 @Etheirystech if it doesn't work yet, create a issue about that (only for chrome profiles avoid adding many thing on one issue)

Still doesn't work on 2.0.5.
![image](https://github.com/user-attachments/assets/c140b89c-5abf-4942-a8e3-edf2f0832582)
